### DISCUSSION TITLE
Why does my battery still charge 1% a day when over the charge limit?

### DISCUSSION THREAD

**Discussion by andrewcrook** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#645] [Q&A] [answered]
When I shutdown I turn the power off at the wall.

MacBook Pro (13-inch, M1, 2020)
macOS 12.3 (21E230)
AlDente Pro 1.15.2 (Lifetime License)

<img width="83" alt="image" src="https://user-images.githubusercontent.com/803618/160603022-be1a4dc7-2c93-44fb-a1e1-91a0eb6d8ac8.png">

<img width="429" alt="image" src="https://user-images.githubusercontent.com/803618/160602944-aa8d70ae-b57f-4381-bdf4-af726666a556.png">

<img width="432" alt="image" src="https://user-images.githubusercontent.com/803618/160601841-31e4b6d0-765a-47c3-ba3e-805203c0a118.png">

**Comment by andrewcrook** [comment]
oh yes I have disabled macOS’s battery optimisation BTW

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @andrewcrook,

My guess would be that there is always a little bit of charging when you turn off your MacBook and shortly after turn off power. The same when you turn it on again. This could explain why it keeps charging just a little bit every day.

Cheers,
Matthias

**Comment by andrewcrook** [comment]
@MatthiasKerbl 

yes that’s what I am thinking. 
I am just a bit surprised that it charge 1% in such a short amount of time.

**Reply by andrewcrook** [reply]
>  guess if you just used standby instead of shutting the system down, that would prevent the unwanted charging.

Yes that’s what we determine the M1 power system is quite different. It resets on boot.

**Reply by andrewcrook** [reply]
> Li-Ion

Li-Poly*
